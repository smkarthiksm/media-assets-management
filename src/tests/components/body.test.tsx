import '@testing-library/jest-dom';
import { renderWithProviders } from '../test-utility';
import BodyComponent from '../../components/body/body';

describe('BodyComponent', () => {
  test('should render', () => {
    const { container } = renderWithProviders(<BodyComponent />);
    expect(container).toMatchSnapshot();
  });
});
