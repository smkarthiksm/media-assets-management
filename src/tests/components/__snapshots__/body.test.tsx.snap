// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AllFilesHeaderComponent should render 1`] = `
<div>
  <div
    class="body-container h-100"
  />
</div>
`;

exports[`BodyComponent should render 1`] = `
<div>
  <div
    class="body-container h-100"
  />
</div>
`;
